Makefile

```diff
@@ -56,7 +81,7 @@ ingest-restart:
 	$(COMPOSE) up -d --no-deps --remove-orphans --force-recreate ingest-file convert-document
 
 dev: 
-	pip install -q bump2version babel jinja2
+	pip install -q -r requirements-dev.txt
```

Comment:
Not directly related to this PR: Is there any reason against changing this to `pip3`? Some people might not have `pip` aliased accordingly.

Reply:
Yes, absolutely. `python3 -m pip install ...` is recommended by the python community nowadays so that `pip` uses the right interpreter. https://snarky.ca/why-you-should-use-python-m-pip/